appservice/src/TunnelProxy.ts

```diff
@@ -201,8 +201,9 @@ export class TunnelProxy {
     }
 
     private async checkTunnelStatus(context: IActionContext): Promise<void> {
+        const publishingUserName: string = nonNullProp(this._publishCredential, 'publishingUserName');
```

Comment:
Minor, but a change due to potentially null `this._publishCredential.publishingUserName`.